public/docs/_examples/tutorial/dart/.gitignore

```diff
@@ -0,0 +1,22 @@
+# Files and directories created by pub
```

Comment:
Are you sure? I saw a .gitignore in the the ts tutorial:

![image](https://cloud.githubusercontent.com/assets/1035299/12304347/5a56557e-ba2f-11e5-8bd1-bbb68c0c7e79.png)